src/main/java/life/qbic/App.java

```diff
@@ -9,6 +9,7 @@
 import life.qbic.model.download.AuthenticationException;
 import life.qbic.model.download.ConnectionException;
 import life.qbic.model.download.QbicDataDownloader;
+import org.apache.commons.lang3.ObjectUtils;
```

Comment:
Hi @Aline-9 
I think this import is not used, correct?
In IntelliJ, you can easily format your code and optimize the imports. Please remove this import 😄